apps/condo/domains/common/utils/helpers.ts

```diff
@@ -25,7 +29,84 @@ export const getFiltersFromQuery = <T>(query: ParsedUrlQuery): T | Record<string
     }
 }
 
-export const preciseFloor = (x: number, precision?: number) => {
-    precision = precision === undefined ? DEFAULT_WIDTH_PRECISION : precision
+export const preciseFloor = (x: number, precision: number = DEFAULT_WIDTH_PRECISION) => {
     return Math.floor(x * Math.pow(10, precision)) / 100
 }
+
+export type TRecordWithAddressDetails = {
```

Comment:
Is it supposed to be used for `Property` itself, which has `address` field, not `property.address`?